docs/source/notebooks/sampling_conjugate_step.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">%config InlineBackend.figure_format = 'retina'
RANDOM_SEED = 8927
np.random.seed(RANDOM_SEED)
az.style.use("arviz-darkgrid")
</pre><p>needs to be in a second cell (because of matplotlib sets defaults)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Yup - got it.</p>